03.bowling/y-bowling.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+split_score = score.split(',')
```

Comment:
配列なので複数形にした方がわかりやすいです。
あと、splitは動詞なので、split_scoreだと「スコアを分割する」というメソッドに見えます。
https://qiita.com/jnchito/items/459d58ba652bf4763820

splitした、という意味なら splited_scores みたいな名前になりますが、そもそもここはシンプルに

```suggestion
scores = ARGV[0].split(',')
```

で良さそうです

Reply:
なるほどです。
こういう単純そうに感じることでも言われてようやく気が付きます。。🥲
思考の柔軟性がたりないのでしょうか、、

Reply:
> 思考の柔軟性がたりないのでしょうか、、

最初から完璧に書ける人はいないので大丈夫です！
僕なんか20年プログラマやってますからね。単純に経験値だけ見てもきっと雲泥の差があると思います。
（同じコードを見ても、jepeさんには見えない何かが僕にはたくさん見えてるはず）

過去に同じようなテーマでブログを書いたので参考にしてみてください。
https://blog.jnito.com/entry/2020/08/24/083030

リーダブルコードやCODE COMPLETEみたいに「読みやすくてわかりやすいコードの書き方」に特化した技術書もあるので、そうした本を読んでみるのもお勧めです！